cmd/swarm/upload.go

```diff
@@ -36,22 +36,67 @@ import (
 	"gopkg.in/urfave/cli.v1"
 )
 
+var (
+	MAX_STDIN_SIZE = 1024 * 1000 * 100
+)
+
+func swapStdin() (size int, path string) {
+	b := make([]byte, 1024)
+	size = 0
+
+	td, err := ioutil.TempDir("", "swarm-stdin")
```

Comment:
What I meant by "leaked" was that when this program exits, the temporary directory created here will not be deleted, so multiple invocations of the program will each "leak" the directory. Does that make sense?